### ISSUE TITLE
train_uds_duration.sh FileNotFoundError: '../../data/test/cached_RobertaTokenizer_128_test-temporal-duration-data.tsv_hypothesis_only_False.lock'

### ISSUE THREAD

**Issue by waterhall** [sidsvash26/temporal_nli#1]
Hi, I am running the train_uds_duration.sh script but I get this error.

```Traceback (most recent call last):
  File "run_finetuning_nli_2gpus.py", line 314, in <module>
    main()
  File "run_finetuning_nli_2gpus.py", line 273, in main
    tokenizer=tokenizer)
  File "/home/waterhall/temporal_nli/src/models/roberta/glue_data_huggingface.py", line 100, in __init__
    with FileLock(lock_path):
  File "/home/waterhall/.local/share/virtualenvs/temporal_nli-5yUdJwgv/lib/python3.7/site-packages/filelock.py", line 323, in __enter__
    self.acquire()
  File "/home/waterhall/.local/share/virtualenvs/temporal_nli-5yUdJwgv/lib/python3.7/site-packages/filelock.py", line 271, in acquire
    self._acquire()
  File "/home/waterhall/.local/share/virtualenvs/temporal_nli-5yUdJwgv/lib/python3.7/site-packages/filelock.py", line 384, in _acquire
    fd = os.open(self._lock_file, open_mode)
FileNotFoundError: [Errno 2] No such file or directory: '../../data/test/cached_RobertaTokenizer_128_test-temporal-duration-data.tsv_hypothesis_only_False.lock'
```
Strangely, I have all the 

> cached_RobertaTokenizer_128_<dev/train/test>-temporal-duration-data.tsv_hypothesis_only_True
> cached_RobertaTokenizer_128_<dev/train>-temporal-duration-data.tsv_hypothesis_only_False

files and lock files.

The only one missing is:
> cached_RobertaTokenizer_128_test-temporal-duration-data.tsv_hypothesis_only_False.lock 
> cached_RobertaTokenizer_128_test-temporal-duration-data.tsv_hypothesis_only_False

I can't figure out what's causing this and would appreciate any help.
Thank you!

**Comment by sidsvash26** [comment]
Will look at this today.

**Comment by sidsvash26** [comment]
@waterhall  This issue was due to a typo in the location of test directory which is now fixed in f376ce4f0708c7a518d9d8568afc21c15a516e02
Since you've already run the first part of the `train_uds_duration.sh` script (`hypothesis_only=True` model), you can directly run the command starting from [Line 21]( https://github.com/sidsvash26/temporal_nli/blob/main/src/models/roberta/train_uds_duration.sh#L21) which will run the `hypothesis_only=False` model. That should save time.
Really sorry about the delay.
